azure-pipelines.yml

```diff
@@ -37,6 +37,7 @@ trigger:
   branches:
     include: 
     - master
```

Comment:
Should we remove the master branch?

Reply:
This is updated to remove master in a later step after changing the default branch and recreating the DARC subscriptions.

Reply:
Actually, I'm going to disable the PR builds entirely so all PRs retargeted to main don't queue up PR builds simultaneously.